ui/components/molecules/M008-Popper/index.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import createPropTypes from 'json-schema-prop-types'
+import { withStyles, Paper } from '@material-ui/core'
+
+import Popper from '@material-ui/core/Popper'
+
+import schema from './schema'
+import styles from './styles'
+
+const M008 = props => {
+  return (
+    <div>
+      <Popper
+        open
```

Comment:
this is basically: open={true} I think

which means this can never be closed

there are two ways of solving this:

1. pass down each prop to popper using the spread operator
{...props}

2. pass down each individual prop
open={props.open}

or better yet if you have put const {open} = props at the top

open={open}

personally I prefer option 1. but it needs to be used carefully :) this is a good time to use it.
https://zhenyong.github.io/react/docs/jsx-spread.html#spread-attributes

Reply:
When I used the open={true} and open={props.open} i have this error 

""standard" found some errors. Please fix them and try committing again.
  /Users/rvg/_projects/cobuy/cobuy-atomic/ui/components/molecules/M008-Popper/stories.js:25:7: Value must be omitted for boolean attributes"

i will try other options..

Reply:
following up on our call - those are completely different things

`open` or `open={true}` are the same thing and basically are saying the variable 'open' is assigned true

`open={props.open}`

or if you have cast your variables out of props into consts 

`open={open}`

are both assigning the future value of 'porps.open' to 'open' in other words the prop is being passed down as well as maintaining its naming convention.